docs/release-notes/8.18.asciidoc

```diff
@@ -0,0 +1,132 @@
+* Adds a new field to the metrics section of the {elastic-defend} metadata document called `top_process_trees`. This section will contain a list of the top noisy processes on the system, with "noisy" being based on how many events they generate.
+* Allows you to opt into aggregation of network events to reduce CPU usage, I/O, and event sizes. Network events with the same addresses and ports occurring in rapid succession will be combined into fewer aggregate events. Use the `advanced.events.aggregate_network` advanced setting in your {elastic-defend} policy to enable it.
+
+[discrete]
+[[enhancements-8.18.0]]
+==== Enhancements
+* Enables the new inference connector for Automatic Import ({kibana-pull}206111[#206111]).
+* Enables new inference connector in the AI Assistant and Attack Discovery ({kibana-pull}204505[#204505]).
+* Enhances Attack discovery by providing you with additional control over which alerts are included as context to the large language model (LLM) ({kibana-pull}205070[#205070]).
+* Provides APIs for AI Assistant Knowledge Base entries ({kibana-pull}206407[#206407]).
+* Adds the product documentation tool to AI Assistant to ensure product docs are installed and can be properly retrieved ({kibana-pull}199694[#199694]).
+* Introduces support for the future integration of AI Assistant prompts in {kib}. ({kibana-pull}207138[#207138]).
+* Adds audit logging for changes to AI Assistant knowledge base entries ({kibana-pull}203349[#203349]).
+* Adds a service example to the entity store upload page ({kibana-pull}209023[#209023]).
+* Update entity insight badge to open entity flyouts ({kibana-pull}208287[#208287]).
+* Introduces changes to the entity analytics feature to support `event.ingested` as a configurable timestamp field for init and enable endpoints ({kibana-pull}208201[#208201]).
+* Allows you to include closed alerts in risk score calculations ({kibana-pull}201909[#201909]).
+* Turns the `securitySolution:enableVisualizationsInFlyout` <<visualizations-in-flyout,advanced setting>> on by default, which allows you to access the event analyzer and Session View in the **Visualize** tab on the alert or event details flyout ({kibana-pull}211319[#211319]).
+* Reduces the system performance impact of {elastic-defend} file events.
+* Improves {elastic-defend}'s resilience in low memory situations.
+* Updates the {elastic-defend} policy status message to show the {elastic-defend} policy name, revision, and {agent} policy revision.
+* Ensures that the data view selector on the rule creation form shows data view names instead of their defined indices ({kibana-pull}214495[#214495]).
```

Comment:
@approksiu and @yctercero made this change in response to https://github.com/elastic/security-docs/pull/6519#issuecomment-2759463443. The PR is labeled to be skipped for release notes, so let me know if I should link to a different one. Thanks! 

```suggestion
* Ensures that the data view selector on the rule creation form shows data view names instead of their defined indices ({kibana-pull}214495[#214495]).
* Allows rule actions (except for **Summary of alerts** actions that run at a custom frequency) to activate during manual rule runs ({kibana-pull}208072[#208072]).
```

Reply:
Agree it should be release-noted 👍🏽